app/com/linkedin/drelephant/tez/TezMetricsAggregator.java

```diff
@@ -85,23 +85,25 @@ public HadoopAggregatedData getResult() {
 
   private long getMapContainerSize(HadoopApplicationData data) {
     try {
-      long mapContainerSize = Long.parseLong(data.getConf().getProperty(TEZ_CONTAINER_CONFIG));
-      if (mapContainerSize > 0)
-        return mapContainerSize;
-      else
-        return Long.parseLong(data.getConf().getProperty(MAP_CONTAINER_CONFIG));
+      if (data.getConf().containsKey(TEZ_CONTAINER_CONFIG)){
```

Comment:
Thanks for contributing in Dr. Elephant. Can you please clean the code a little bit, with single return statement.